restricted_site_access.php

```diff
@@ -1552,7 +1552,10 @@ public static function get_client_ip_address() {
 			) as $ip ) {
 				$ip = trim( $ip ); // just to be safe.
 
-				if ( filter_var( $ip, FILTER_VALIDATE_IP, FILTER_FLAG_NO_PRIV_RANGE | FILTER_FLAG_NO_RES_RANGE ) !== false ) {
+				/** Hook to filter IP flags. */
+				$filter_flags = apply_filters( 'rsa_get_client_ip_address_flags', FILTER_FLAG_NO_PRIV_RANGE | FILTER_FLAG_NO_RES_RANGE );
```

Comment:
I think we can use a more explicit filter name here, something link `rsa_get_client_ip_address_fliter_flags`

Reply:
@Sidsector9 thoughts on updated naming ^?

Reply:
Makes sense. I've pushed the updated name.